test/unit/CMakeLists.txt

```diff
@@ -15,6 +15,17 @@ include_directories(${PROJECT_SOURCE_DIR}/src/box)
 include_directories(${PROJECT_SOURCE_DIR}/third_party)
 include_directories(${ICU_INCLUDE_DIRS})
 include_directories(${CURL_INCLUDE_DIRS})
+
+# There are many unit tests that include fiber.h, which may
+# transitively include libunwind.h. It requires two actions:
+#
+# 1. Add the header directory into the search paths.
+# 2. Add the dependency to don't try to find libunwind.h before
+#    it'll appear in given directory.
+if (ENABLE_BACKTRACE AND ENABLE_BUNDLED_LIBUNWIND)
+  include_directories(SYSTEM ${LIBUNWIND_INCLUDE_DIR})
+endif()
```

Comment:
AFAIC, the included directories are inherited by all subdirectories:
> The include directories are added to the [INCLUDE_DIRECTORIES](https://cmake.org/cmake/help/latest/prop_dir/INCLUDE_DIRECTORIES.html#prop_dir:INCLUDE_DIRECTORIES) directory property for the current CMakeLists file. They are also added to the [INCLUDE_DIRECTORIES](https://cmake.org/cmake/help/latest/prop_tgt/INCLUDE_DIRECTORIES.html#prop_tgt:INCLUDE_DIRECTORIES) target property for each target in the current CMakeLists file.[^1]

This snippet added to the beginning of this files yields the following result:
```cmake
get_property(INCLUDE_DIRECTORIES DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR} PROPERTY INCLUDE_DIRECTORIES)
message("INCLUDE_DIRECTORIES=${INCLUDE_DIRECTORIES}")
```
```console
INCLUDE_DIRECTORIES=/Users/georgiy.lebedev/Work/tarantool/src;/Users/georgiy.lebedev/Work/tarantool/build-release/src;/Users/georgiy.lebedev/Work/tarantool/src/lib;/Users/georgiy.lebedev/Work/tarantool/src/lib/small/include;/Users/georgiy.lebedev/Work/tarantool/src/lib/small/third_party;/Users/georgiy.lebedev/Work/tarantool/src/lib/core;/Users/georgiy.lebedev/Work/tarantool/third_party;/usr/local/opt/openssl@3/include;/Users/georgiy.lebedev/Work/tarantool/third_party/xxHash;/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX12.3.sdk/usr/include;/Users/georgiy.lebedev/Work/tarantool/third_party/luajit/src;/Users/georgiy.lebedev/Work/tarantool/build-release/third_party/luajit/src;/Users/georgiy
```

Perhaps, we should rather move all `include_directories` commands from [`src/CMakeLists.txt`](https://github.com/tarantool/tarantool/blob/05002edf80d44a4a82da8be0d5e7bcda5349d47a/src/CMakeLists.txt#L6-L18) to [`CMakeLists.txt`](https://github.com/tarantool/tarantool/blob/05002edf80d44a4a82da8be0d5e7bcda5349d47a/CMakeLists.txt#L293-L302)?

[^1]: https://cmake.org/cmake/help/latest/command/include_directories.html

Reply:
I thought on it, but I'm afraid to make major changes in our build scripts without some consistent vision on it. IOW, I would proceed this way for all bundled libraries if we'll be sure that it is the right thing to do. But I wouldn't make it as a fix of particular build problem.